osv/sources.py

```diff
@@ -52,6 +53,8 @@ def repo_path(repo):
 
 
 class NoDatesSafeLoader(yaml.SafeLoader):
```

Comment:
there's obviously sometime about YAML parsing that I'm ignorant about here. Can you add some sort of hint for readers like me?

Reply:
Added more detailed docstring

Reply:
Thanks, that was very helpful